atomkraft/_version.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.0.9"
```

Comment:
We can do the following to read the version number from `pyproject.toml`:
```
import importlib.metadata
__version__ = importlib.metadata.version("atomkraft")
```

Reply:
I don't like `importlib.metadata` because it uses installed distribution, which is not the same as the source code. 

If I installed an older version of `atomkraft` and used `importlib.metadata` to fetch the version string, the current version test would fail on my machine.

Reply:
Good point. I didn't know exactly how this library works. So ignore my suggestion and go ahead with your code.